source/StigData/Processed/Ubuntu-18.04-2.3.xml

```diff
@@ -0,0 +1,6563 @@
+      <LegacyId>V-100681</LegacyId>
+      <OrganizationValueRequired>True</OrganizationValueRequired>
+      <OrganizationValueTestString>the following statement is true when leveraging the correct nxFileLine ContainsLine format: If the value of the "disk_full_action" option is not "SYSLOG", "SINGLE", or "HALT", or the line is commented out, this is a finding. </OrganizationValueTestString>
+      <RawString>Verify the Ubuntu operating system takes the appropriate action when the audit storage volume is full. 
+
+Check that the Ubuntu operating system takes the appropriate action when the audit storage volume is full with the following command:
+
+# sudo grep disk_full_action /etc/audit/auditd.conf
+
+disk_full_action = HALT
+
+If the value of the "disk_full_action" option is not "SYSLOG", "SINGLE", or "HALT", or the line is commented out, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-219238" severity="medium" conversionstatus="pass" title="SRG-OS-000064-GPOS-00033" dscresource="nxFileLine">
+      <ContainsLine>-a always,exit -F path=/bin/su -F perm=x -F auid&gt;=1000 -F auid!=-1 -k privileged-priv_change</ContainsLine>
+      <Description>&lt;VulnDiscussion&gt;Without generating audit records that are specific to the security and mission needs of the organization, it would be difficult to establish, correlate, and investigate the events relating to an incident or identify those responsible for one.
+
+Audit records can be generated from various components within the information system (e.g., module or policy filter).&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+      <DoesNotContainPattern>#\s*-a\s*always,exit\s*-F\s*path\s*=\s*/bin/su\s*-F\s*perm\s*=\s*x\s*-F\s*auid&gt;\s*=\s*1000\s*-F\s*auid!\s*=\s*-1\s*-k\s*privileged-priv_change</DoesNotContainPattern>
+      <DuplicateOf />
+      <FilePath>/etc/audit/rules.d/audit.rules</FilePath>
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>V-100703</LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>Verify if the Ubuntu operating system generates audit records when successful/unsuccessful attempts to use the "su" command occur.
+
+Check the configured audit rules with the following commands:
+
+# sudo auditctl -l | grep '/bin/su'
+
+-a always,exit -F path=/bin/su -F perm=x -F auid&gt;=1000 -F auid!=-1 -k privileged-priv_change
+
+If the command does not return lines that match the example or the lines are comme
```

Comment:
"removexattr" is an update from N-1. We need to test if the doesnotcontainPattern  does the same as the new grep command